README.md

```diff
@@ -1,2 +1,2 @@
-# review-projects-from-trails
-this is for pull request review 
+# Review-of-past-projects
+This repo is made for reviewing of my Past Projects
```

Comment:
no discriptive readme file

Reply:
Your README file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present 👌 Kindly, update it 👌